server/database.go

```diff
@@ -213,38 +212,6 @@ func (s *Server) registerDatabaseRoutes(g *echo.Group) {
 				if !s.feature(api.FeatureMultiTenancy) {
 					return echo.NewHTTPError(http.StatusForbidden, api.FeatureMultiTenancy.AccessErrorMessage())
 				}
-
```

Comment:
Just remove the full if block starting at l212?

Reply:
Done. Was thinking of restricting tenant mode on an expired license. I guess it’s okay to transfer databases across projects as long as no issues are created.